pkgs/development/python-modules/pbxproj/default.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  pytestCheckHook,
+  docopt,
+  openstep-parser,
+}:
+
+buildPythonPackage rec {
+  pname = "pbxproj";
+  version = "4.2.0";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```

this will use pypa infra to build packages

https://nixos.org/manual/nixpkgs/unstable/#buildpythonpackage-parameters